docs/UserGuide.md

```diff
@@ -147,6 +147,19 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Deleting a Meeting : `delMeeting`
+
+Delete a meeting from MyInsuRec.
```

Comment:
Let's follow the convention of using plural forms for the action, for example 'Deletes' instead of 'Delete'.

Reply:
Alright